pkg/storage/stores/indexshipper/compactor/compactor.go

```diff
@@ -98,6 +101,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.DeletionMode, "boltdb.shipper.compactor.deletion-mode", "disabled", fmt.Sprintf("Deletion mode. Can be one of %v", strings.Join(deletion.AllModes(), "|")))
 	cfg.CompactorRing.RegisterFlagsWithPrefix("boltdb.shipper.compactor.", "collectors/", f)
 	f.BoolVar(&cfg.RunOnce, "boltdb.shipper.compactor.run-once", false, "Run the compactor one time to cleanup and compact index files only (no retention applied)")
+	f.IntVar(&cfg.TablesToCompact, "boltdb.shipper.compactor.tables-to-compact", 0, "The number of most recent tables to compact in a given run. Default: all")
+	f.IntVar(&cfg.YoungestTableToCompact, "boltdb.shipper.compactor.youngest-table", 0, "Compact only tables older than the n-th youngest.")
```

Comment:
What do you think about naming it like this? Makes it clearer. Any other suggestions are welcome.
```suggestion
	f.IntVar(&cfg.SkipLatestNTables, "boltdb.shipper.compactor.skip-latest-n-tables", 0, "Skip compacting latest N tables")
```

Reply:
Thought a bit about how to phrase this. I think your suggestion is better than anything I can come up with.